packages/components/src/internal/components/domainproperties/assay/AssayPropertiesInput.tsx

```diff
@@ -328,6 +328,26 @@ export class AutoLinkDataInput extends React.PureComponent<InputProps, AutoLinkD
     }
 }
 
+export class AutoLinkCategoryInput extends React.PureComponent<InputProps> {
```

Comment:
Have you used React FunctionalComponents much yet? If not, this would be a good one to convert as a simple example as it just takes in two props and doesn't really do much logic. See other usages of `FC<Props>`.

Reply:
Done!